plan/2.0 Expand Training.md

```diff
@@ -38,338 +16,519 @@
+  - SKF Operations team will integrate security test automation in the CI/CD pipeline
+    - SAST
+    - SCA
+    - Start a bug bounty program (TBD what platform)
+- **OPS4**: Infra Maintenance
+
+  - Cloud Operations
+  - Make sure that SKF Cloud services remain operational.
+  - Make sure that the SKF Platform runs within its allocated budget.
+
+- **OPS5**: Services
+  - Make sure that SKF System services remain operational and up-to-date.
+- **OPS6**: Information gathering
+  - APM
+  - Analytics
+  - Security and event monitoring
+
+### Steward
+
+- **OPS7**: API Development
+  - SKF Operations will architect, develop, and maintain all SKF APIs offered.
+- **OPS8**: UI/UX
+  - SKF Operations will design, maintain, and continuously improve the SKF Platform's UI/UX.
+- **OPS9**: Feedback
+  - SKF Operations will ensure that community feedback is consistently prioritized.
+  - SKF Operations shall work with the Education SIG as it's primary channel for community feedback.
+- **OPS10**: Content quality
 
-### Work Scope
+  - SKF Operations shall work with the content providers/donators for improving the source materials when possible gaps in materials are identified.
+  - SKF Operations shall work with the content providers/donators to make sure the hands-on labs are correctly being set up and are the level of quality we desire.
+
+- **OPS11**: Organize
+  - Bi-weekly public meetings where we prioritise the work to be executed
+  - Executive meetings (Stakeholders, Team Lead, Architect) for roadmap & strategic discussions
+  - Active in the Slack, Gitter, Discord channels to address the questions of the users related to the SKF features, material information and Labs. Live support channel.
+
+## Team
+
+### Project management
+
+- _Project Lead - Mission, vision, strategy_
+- _Project Lead - Quality Assurance_
+- _Project Lead - Architect_
+
+### Lead Developer
+
+- Full Stack Developer
+
+### Development Team
+
+- Full Stack Developer
+- Full Stack Developer
+
+### Design Team
+
+- UX Designer
+
+### Platform Team
+
+- DevOps Engineer
+
+### Time & Resource Estimate
+
+> **TOTAL**: 8 x Full-time for year 1
+> **TOTAL**: 65k Infrastructure cost
+
+---
 
 ## 2.2 Goal: Refactor 'Lab' contribution procedures
 
-### Explanation: We want to make new contributions to the SKF Labs as smooth as possible but also make sure we have high quality materials being pushed
+### Explanation: We want to make new contributions to the SKF Labs as smoothly as possible but also make sure we have high-quality materials being pushed
 
-- [ ] **M1**: Review current 'Lab' contribution procedures
-  - [ ] **M1.1**: Publish 'Lab/Write-up Style Guide'
+### Key Steps/Milestones
 
-#### Time & Resource Estimate
+- **M1**: Publish updated 'Lab' contribution procedures
+  - [ ] Review currently established 'Lab' contribution procedures
+  - [ ] Refactor 'Lab' contribution procedures into a 'Lab/Write-up Style Guide' document
+  - [ ] Publish 'Lab/Write-up Style Guide' document
```

Comment:
Would also recommend adding the creation of a "content guide" (unless this is expected to be included in the style guide). The lab write up and style guide may cover the mechanics of developing the lab. However there may also be reason to define a minimum content quality measure to ensure the same standard of material value is being applied across all labs.  i.e. all terms in a lab provide a definition at their first introduction that does not include the term itself as part of the definition. Explanations of concepts that leverage metaphors should be commonplace metaphors that are not culturally or hobby-ist-dependent for context.

Reply:
@TheFoxAtWork Great suggestion. I've update the language to reflect my understanding. How does that look to you?

Reply:
LGTM